features/steps/fixture-loading.py

```diff
@@ -27,6 +27,11 @@ def check_second_fixtures(context):
 
 
 @then(u'the sequences should be reset')
-def check_reset_seqeunces(context):
+def check_reset_sequences(context):
     context.test.assertEqual(BehaveTestModel.objects.first().pk, 1)
     context.test.assertEqual(BehaveTestModel.objects.last().pk, 2)
+
+
+@then(u'multi_db should be enabled')
+def check_multi_db(context):
+    assert context.test.multi_db
```

Comment:
I'm actually a friend of plain `assert`, but I fear we need to use boilder-plate `assertEquals` to stay consistent with the rest of the code.  :disappointed:

Reply:
Very good point. I have updated this now.